pkg/scuzz/control.go

```diff
@@ -15,17 +15,72 @@ type Request interface {
 	String() string
 }
 
+// Response is an interface for scuzz responses.
+// It can return an error indicating the drive
+// operation did not go well, a raw status block,
+// or a string suitable for human consumption.
+//
+// Note the error from the Operate is a transport
+// level error and the error in the Response
+// describes target errors. For example,
+// an error return from Operate might indicate
+// permission denied or a similar issue; an error
+// in the Response might indicate the drive had
+// some issues with the request. Distinguishing
+// the two types is complicated by the fact that
```

Comment:
because transport and target mean different things. not being able to start a command is one thing; the command returning an error from the target is something else altogether. This is similar to, e.g., the difference between not being able to write a socket and getting back an error from the thing you were talking to; or the error you get when you receive an rpc vs. the error contained in the response. I don't see the sense in merging them because to see what happened you've got to pull it apart anyway.

Reply:
The alternative I provided still let's you distinguish?

Reply:
I see.

Is there any way you could make the comment more concise to say that?

Reply:
yep. I expect at some point we're going to want to unpack the specific driver error and possibly take other action. The status block is a rich source of error information and might point us to doing other things. So transport error vs. drive error is a big thing. Since the only place this is currently visible is hdparm I'd like to leave it as is.

Reply:
I may end up changing it later (I like your idea) but am not quite ready to now.

Reply:
actually I figured people would want a bit more explanation, hence the long comment :-)

Reply:
I amended the comment a bit with a short sentence and the rest is now under a tl;dr :-)